lib/sendmail_handler.ml

```diff
@@ -21,9 +21,11 @@ let rdwr =
   { Colombe.Sigs.rd =
       (fun { ic; _ } bytes off len ->
         let open Lwt.Infix in
-        let res = Lwt_io.read_into ic bytes off len >>= function
-           | 0 -> Lwt.return `End
-           | len -> Lwt.return (`Len len)
+        let res =
+          Lwt_io.read_into ic bytes off len
+          >>= function
+          | 0 -> Lwt.return `End
+          | len -> Lwt.return (`Len len)
```

Comment:
This formatting looks a bit weird to me. I would prefer the old one.

Reply:
@mikonieminen The formatting was changed by the ocamlformat settings.
Adding `break-infix-before-func = false` to .ocamlformat would result in your expected formatting.
Should I add this as well?

Reply:
Imo we should stick to the vanilla ocamlformat config (with the janestreet profile).